### ISSUE TITLE
GameGuru MAX - Request - Slider for Height Map Strength

### ISSUE THREAD

**Issue by MonkeyFrogStudio** [TheGameCreators/GameGuruRepo#4538]
Started to play with height maps for objects in MAX. Here's an initial test with an old texture:

![stone_floor](https://user-images.githubusercontent.com/36735507/233901399-72260ca8-68f3-4efc-844c-b02747d79e6f.png)

The above image is a single, flat plane (one quad face/two tri faces - no subdivision, etc.). So, we can see that height maps work pretty well overall. Nice job, Lee! Thank you!

My height map here is too strong. As a result, there are some fairly severe artifacts:

![stone_floor2](https://user-images.githubusercontent.com/36735507/233901584-eee2c234-112b-4373-819c-adcd0bebfc1c.png)

My only option with the current edition of MAX would be to redo the heigh map, test, and see if I get it right. What I would like to propose is a slider for Height Maps for objects to increase/decrease the strength/effect of the map. This would be just like the sliders we have for normal maps, etc. That way, fixing these artifacts would be as simple as sliding a slider instead of having to regenerate a height map or bringing it into paint software to alter it, then bringing it back into MAX to test and see if you get the desired results. With a slider, you can visually and instantly see the affect.

**Comment by davetgc** [comment]
Also requested here :  #5200

**Comment by LeeBamberTGC** [comment]
@MonkeyFrogStudio Perhaps something like:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/d62d9f12-1df5-4054-b33a-dfd847f21fa9)

**Comment by LeeBamberTGC** [comment]
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/8c9e49d5-b8da-4332-b118-f48515f021f6)

**Comment by LeeBamberTGC** [comment]
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/35840583/5bc5f3ec-8ee4-4882-a25d-4d2d14913451)

**Comment by MonkeyFrogStudio** [comment]
@LeeBamberTGC Yes, exactly like that ... just like the other sliders for Roughness, Metalness, and Normal. Thanks!
